### PR TITLE
feat(oui-inline-adder): inline-adder component incorporation

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
UU-835
The inline-adder component has been implemented.

**Linked to**
https://github.com/ovh-ux/ovh-ui-angular/pull/179
https://github.com/ovh-ux/ovh-ui-kit-documentation/pull/100

### CODE REVIEW

File: packages/oui-inline-adder/README.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+## Usage
+
+### Normal
+
+```html:preview
+<div class="oui-inline-adder">
+  <div class="oui-inline-adder__item">
+    <div class="oui-inline-adder__row">
+      <div style="flex: 1 1 auto; padding: 1rem; padding-bottom: 0;">
```
Comment (AxelPeter) [inline packages/oui-inline-adder/README.md]: No inline style in our examples. Why not create a class these col/cell (with a modifier if necessary) ? :)

Comment (ganeshkumar1989) [reply packages/oui-inline-adder/README.md]: This is done. Thanks for the suggestion :)

File: packages/oui-inline-adder/inline-adder.less [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    display: flex;
+    flex-direction: row;
+    flex: 1 1 auto;
+    padding: rem-calc(10px);
+
+    &:not(:nth-child(1)) {
+      border-top: 1px solid @oui-color-silver;
+    }
+
+    > * {
```
Comment (AxelPeter) [inline packages/oui-inline-adder/inline-adder.less]: We should avoid using `*` in our styles.
Prefer using classname, maybe with something like that for example:

```less
&__cell {
    padding: 1rem 1rem 0 1rem;

    &_extended { flex: 1 1 auto; }
}
```
to be used like this:
```html
<div class="oui-inline-adder__row">
    <div class="oui-inline-adder__cell oui-inline-adder__cell_extended">
        [...]
    </div>
    <div class="oui-inline-adder__cell">
        [...]
    </div>
</div>
```

File: packages/oui-inline-adder/inline-adder.less [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      padding: 1rem 1rem 0 1rem;
+    }
+  }
+
+  &__flexed-element {
```
Comment (AxelPeter) [inline packages/oui-inline-adder/inline-adder.less]: See my upper comment

File: packages/oui-inline-adder/inline-adder.less (hunk 2 above)
Comment (ganeshkumar1989) [reply packages/oui-inline-adder/inline-adder.less]: Thanks for the idea :). New cell and auto-grow classes have been created

File: packages/oui-inline-adder/inline-adder.less (hunk 3 above)
Comment (ganeshkumar1989) [reply packages/oui-inline-adder/inline-adder.less]: New classes created.

File: packages/oui-inline-adder/README.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+## Usage
+
+### Normal
+
+```html:preview
+<div class="oui-inline-adder">
+  <div class="oui-inline-adder__item">
+    <div class="oui-inline-adder__row">
+      <div class="oui-inline-adder__cell oui-inline-adder__cell-auto-grow">
```
Comment (neolitec) [inline packages/oui-inline-adder/README.md]: `auto-grow` is a modifier so it should be `_auto-grow`.

Comment (ganeshkumar1989) [reply packages/oui-inline-adder/README.md]: This is done :)

**Comment by jleveugle** [comment]
Hello @ganeshkumar1989,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by jleveugle** [comment]
Hello @ganeshkumar1989,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**